mssql-mixin/rows.libsonnet

```diff
@@ -0,0 +1,17 @@
+local g = import './g.libsonnet';
+
+// Use g.util.grid.wrapPanels() to import into custom dashboard
+{
+  new(this): {
+    database:
+      [
+        g.panel.row.new('Database'),
+        this.grafana.panels.databaseWriteStallDurationPanel
+        + g.panel.timeSeries.gridPos.withW(12),
+        this.grafana.panels.databaseReadStallDurationPanel
+        + g.panel.timeSeries.gridPos.withW(12),
+        this.grafana.panels.transactionLogExpansionsPanel
+        + g.panel.timeSeries.gridPos.withW(24),
+      ],
+  },
```

Comment:
best if row panels are added not as array but inside
`g.panel.row.withPanels()` function. This way it is possible to toggle rows collapsed/uncollapsed.

example:
https://github.com/grafana/jsonnet-libs/blob/master/kafka-observ-lib/rows.libsonnet#L25-L33.
and then:
https://github.com/grafana/jsonnet-libs/blob/master/kafka-observ-lib/dashboards.libsonnet#L33-L52